packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -127,7 +127,10 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     if (event is PointerMoveEvent) {
       final VelocityTracker tracker = _velocityTrackers[event.pointer];
       assert(tracker != null);
-      tracker.addPosition(event.timeStamp, event.position);
+      // PointerMoveEvent from platform's gesture up events are positionally redundant
+      // vs the last PointerMoveEvent. Do not use in velocity calculation.
+      if (event.sourceEventType != PointerChange.up)
+        tracker.addPosition(event.timeStamp, event.position);
```

Comment:
So what is the value of the coordinate in the last touchesEnded? (On iOS in general, I mean, not in Flutter.)